src/magic_folder/test/cli/test_magic_folder.py

```diff
@@ -592,6 +652,61 @@ def test_poll_interval(self):
             )
         self.assertEqual(str(ctx.exception), "--poll-interval must be a positive integer")
 
+    def test_poll_interval_negative(self):
+        with self.assertRaises(usage.UsageError) as ctx:
+            parse_cli(
+                "add",
+                "--name", "test",
+                "--author", "test",
+                "--poll-interval=-1",
+                self.temp.path
+            )
+        self.assertEqual(str(ctx.exception), "--poll-interval must be a positive integer")
+
+    def test_poll_interval_zero(self):
+        with self.assertRaises(usage.UsageError) as ctx:
+            parse_cli(
+                "add",
+                "--name", "test",
+                "--author", "test",
+                "--poll-interval=-0",
```

Comment:
And what happens with plain `0` here?  I expect a `UsageError` to be thrown?

Reply:
I think this was a bad copy-paste from the `-1` (where I changed the `1` and didn't remove the `0`. Fixed.